testing/market/src/main.rs

```diff
@@ -156,14 +149,8 @@ fn main() {
         .execute_message(message, ApplyKind::Explicit, 100)
         .unwrap();
 
-    if res.msg_receipt.exit_code.value() != 33 {
-        dbg!(&res);
-    }
-
-    // FIXME : "actor error"
+    // FIXME : "no such deal"
     assert_eq!(res.msg_receipt.exit_code.value(), 33);
-    //assert_eq!(hex::encode(res.msg_receipt.return_data.bytes()), "586408c379a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000136163746f72206572726f7220636f646520313700000000000000000000000000");
```

Comment:
We should here at least leave the raw error response. It will always return 33 as exit code when there is an error. We won't be able to know if the actual error is "no such deal" or any other error until we parse the response otherwise.

Reply:
We are going to fix  the "no such deal" error. It was supposed to be temporary. The test is supposed to return 0 as an exit code otherwise we are no testing the return answer.